website/blog/2020-05-18-rn4mupdadates.md

```diff
@@ -0,0 +1,68 @@
+
+As usual this means that our getting started guides and general docs will also be updated to reflect these changes.
+
+## React Native for macOS
+
+![react-native-mac](assets/eloy_rn4m_preview_full.png)
+
+One of our biggest announcements recently is our addition and full support of the React Native for macOS extension! We are super excited to be developing React Native support for macOS along side our Windows effort.
+
+Although still a fork, we have an official [npm package out for React Native for macOS](https://www.npmjs.com/package/react-native-macos) and a [repo available](https://github.com/microsoft/react-native-macos) to the public to follow and contribute to - we hope to see some really cool React Native apps out there in the future that can now truly run everywhere.
+
+## New Module Support!
+
+We've begun addressing the Windows community module support gap - starting with two contributions from us! The [**WebView**](https://github.com/react-native-community/react-native-webview) and [**Camera**](https://github.com/react-native-community/react-native-camera) modules.
+
+On top of those initial two, we're looking at a whole bunch of other ones we know are awesome and essential to have when building a React Native app in general. Check out the full list of what we're tracking and planning to tackle next on our [GitHub Project Board](https://github.com/microsoft/react-native-windows/projects/23) for modules.
+
+We're also introducing release stages!
+
+### WebView
+
+![react-native-webview-module](assets/final_demo_gif.gif)
+
+The WebView module sets you up with the barebones web hosting tech that's available nativly on the device you're targetting - as a bonus it works with macOS as well!
+
+### Camera
+
+![react-native-camera-module](assets/react-camera.png)
+
+The Camera module is another big step towards making it easier to give you unfettered access to native APIs.
+
+Just like on mobile, when you define a CameraFeed module, and the device you're running the app on has a camera attached, the component will hookup to the correct native APIs to give you access to the hardware needed to snap sweet pics.
+
+## React Native on the Surface Duo
+
+![react-native-duo-emu](https://devblogs.microsoft.com/surface-duo/wp-content/uploads/sites/53/2020/04/reactnative-emulator-800.png)
+
+React Native runs on the Surface Duo!
+
+This is a little bit to be expected, as React Native works great today with Android, but just running on the new Surface Duo device isn't enough. We've also added support for awareness of the dual-screen device and it's status through shipping two important APIs: **TwoPaneView** and **DeviceInfoModule**.
+
+To learn more about developing on the Surface Duo with React Native, [go check out our latest blog post](https://devblogs.microsoft.com/surface-duo/build-react-native-apps-for-microsoft-surface-duo/) detailing the experience.
+
+## New Release Processes

```

Comment:
Recommend skipping this section at this time. Lets wrap up 0.62 properly and announce the new release process as part of 0.62 stable update. I would also like to have this documented as part of the [versions](https://microsoft.github.io/react-native-windows/versions) page as a long lived document we can point to at the time of update to 0.62

Reply:
removed the releases detail table - will save it for next blog update.